docs/latest/user/egctl.md

```diff
@@ -451,12 +453,176 @@ spec:
           kind: Service
           name: backend
           port: 3000
-          weight: 1
+EOF         value: /thPrefix
```

Comment:
is this right?

Reply:
thanks for catching this, fixed it, ptal

Reply:
hoping https://github.com/envoyproxy/gateway/pull/1175/files lands soon

Reply:
only available for github internal project